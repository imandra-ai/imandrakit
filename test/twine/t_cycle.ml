let str = "61f001" |> CCString.of_hex_exn

let () =
  try
    ignore (Imandrakit_twine.Dump.dump_string str : string);
    assert false
  with exn -> Printf.printf "failed: %s\n%!" (Printexc.to_string exn)
