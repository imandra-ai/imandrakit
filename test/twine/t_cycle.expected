failed: File "src/twine/dump.ml", line 83, characters 6-12: Assertion failed
